pypopquiz/io.py

```diff
@@ -155,6 +156,24 @@ def verify_input(input_data: Dict) -> None:
         }
     }
     jsonschema.validate(input_data, schema)
+
+    # Sets intervals to the full duration if not specified
+    for index, question in enumerate(input_data["questions"]):
+        for sub_type in ("question_video", "question_audio", "answer_video", "answer_audio"):
+            for sub_item in question[sub_type]:
+                source_index = sub_item["source"]
+                if "interval" not in sub_item.keys():
+                    source = question["sources"][source_index]
+                    if "duration" not in source.keys():
+                        raise ValueError("Missing interval for question {:d}'s '{:s}'".format(index, sub_type))
+                    else:
+                        minutes = source["duration"] // 60
```

Comment:
Eventually we could consider using something like time.strptime, but I guess this works for now.

Reply:
Ah yes, maybe we should use something like that later, I also saw you introduced a sub-second unit. Let's fix everything in the code separately later.